pycbc/events/significance.py

```diff
@@ -84,7 +84,7 @@ def count_n_louder(bstat, fstat, dec,
 
     unsort = sort.argsort()
     back_cum_num = n_louder[unsort]
-    return back_cum_num, fore_n_louder
+    return back_cum_num, fore_n_louder, {}
```

Comment:
Hmm, I am not getting what is the logic behind returning an empty dictionary here.  I don't see it getting used downstream so maybe this was a placeholder change that in the end isn't needed?

Reply:
Ah, it looks like this is to keep using the same signature between the time slide n-louder method vs the exp fit one. 
Then add a comment before this line to point out that this is in parallel to n_louder_from_fit

Reply:
As noted in the review, the 'Returns' part of the docstring should mention that an empty dict is also returned